### PR TITLE
ZCS-909: Universal UI, attach interface on the inbox reading pane

### PR DESCRIPTION
**Pull request by kalpeshhpatel**
Changes:
* ZmMimetable.js:
  * Updated icons for different file mime types.
  * Need to handle some of the cases in future added in `TDOD`
* ZmMailMsgView.js:
  * Updated `this._attcMaxSize` variable as per current mockup specs.
  * Updated `ZmMailMsgView.prototype._setAttachmentLinks` method to change markup for attachment section
  * Updated `ZmMailMsgView.getAttachmentLinkHtml` method to add support for showing icons.
* Message.template:
  * Updated template structure as per requirement.
* Updated strings in `ZmMsg.properties` as per requirement.
* Added/updated styles as per requirement in `skin.css` and `zm.css`
* Added/updated properties as per requirement in `skin.properties`
* Added/updated/removes icons as per requirements.

https://jira.corp.synacor.com/browse/ZCS-909

### CODE REVIEW

File: WebRoot/js/zimbraMail/core/ZmMimeTable.js [hunk 1]
```diff
@@ -149,9 +151,10 @@ ZmMimeTable._table[ZmMimeTable.TEXT_XML]			= {desc: ZmMsg.xmlDocument, image: "G
...
 ZmMimeTable._table[ZmMimeTable.VIDEO_WMV]			= {desc: ZmMsg.msWMV, image: "VideoDoc", imageLarge: "VideoDoc_48", dataUri: ZmMimeTable._icons.video};
-ZmMimeTable._table[ZmMimeTable.TEXT_DIRECTORY]		= {desc: ZmMsg.vCard, image: "GenericDoc", imageLarge: "GenericDoc_48", dataUri: ZmMimeTable._icons.generic};
-ZmMimeTable._table[ZmMimeTable.TEXT_VCARD]			= {desc: ZmMsg.vCard, image: "GenericDoc", imageLarge: "GenericDoc_48", dataUri: ZmMimeTable._icons.generic};
-ZmMimeTable._table[ZmMimeTable.TEXT_X_VCARD]		= {desc: ZmMsg.vCard, image: "GenericDoc", imageLarge: "GenericDoc_48", dataUri: ZmMimeTable._icons.generic};
+ZmMimeTable._table[ZmMimeTable.TEXT_DIRECTORY]		= {desc: ZmMsg.vCard, image: "ContactDoc", imageLarge: "ContactDoc_48", dataUri: ZmMimeTable._icons.generic};
+ZmMimeTable._table[ZmMimeTable.TEXT_VCARD]			= {desc: ZmMsg.vCard, image: "ContactDoc", imageLarge: "ContactDoc_48", dataUri: ZmMimeTable._icons.generic};
+ZmMimeTable._table[ZmMimeTable.TEXT_X_VCARD]		= {desc: ZmMsg.vCard, image: "ContactDoc", imageLarge: "ContactDoc", dataUri: ZmMimeTable._icons.generic};
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/core/ZmMimeTable.js]: ContactDoc_48

Comment (kalpeshhpatel) [inline WebRoot/js/zimbraMail/core/ZmMimeTable.js]: Fixed: https://github.com/Zimbra/zm-web-client/pull/20/commits/df543ba2292f60e27250a4e7777189c2a7341d70

File: WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ ZmMailMsgView = function(params) {
 
 	this._scrollWithIframe = params.scrollWithIframe;
 	this._limitAttachments = this._scrollWithIframe ? 3 : 0; //making it local
-	this._attcMaxSize = this._limitAttachments * 16 + 8;
+	this._attcMaxSize = this._limitAttachments * 32 + 8;
```
Comment (tarangkhandelwal) [inline WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js]: what is 32 here? css style value?

Comment (kalpeshhpatel) [inline WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js]: It's height of attachment row item. It was 16px before, now it's 32px.

Thanks

**Comment by kalpeshhpatel** [comment]
Squashed commits and merged. 

Thanks

**Pull request merged**